File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int length = s.length() / 2;

        if (s.length() % 2 == 0) {
            answer = s.substring(length - 1, length + 1);
        } else {
            answer = Character.toString(s.charAt(length));
        }

        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    console.log(Math.ceil(s.length/2) - 1 )
    
    if(s.length % 2 != 0){
        answer = s.charAt(Math.ceil(s.length/2) - 1);
    } else{
        answer = s.charAt(s.length/2 - 1) + s.charAt(s.length/2) ;
    }
    
    return answer;
}